apps/condo/domains/user/schema/SendMessageToSupportService.js

```diff
@@ -61,6 +61,8 @@ const SendMessageToSupportService = new GQLCustomSchema('SendMessageToSupportSer
                 const { data } = args
                 const { dv, sender, text, emailFrom, attachments = [], os, appVersion, lang } = data
 
+                if (!SUPPORT_EMAIL_MOBILE) throw new Error('Wrong server side support email configuration!')
```

Comment:
same check as in line 26

Reply:
It’s a little different. 26 it just a warning. But here is a guard.